### PR TITLE
Migrate dlr to datasets

### PR DESCRIPTION
**Pull request by CarlosEpia**
The class "Calculate_dlr" was created and the script to execute it was moved to the folder "datasets". Additionally, the names of the tables (source/target) are related to the file datasets.yml.

Already tested locally for Schleswig Holstein.

### CODE REVIEW

File: src/egon/data/airflow/dags/pipeline.py [hunk 1]
```diff
@@ -316,12 +321,14 @@
...
-    osmtgmod_pypsa >> calculate_dlr
-    download_data_bundle >> calculate_dlr
-    download_weather_data >> calculate_dlr
+    dlr = Calculate_dlr(
+        dependencies=[osmtgmod_pypsa,
+                      download_data_bundle,
+                      download_weather_data,
+            ]
+    )
+    #download_data_bundle = tasks["data_bundle.download"]
```
Comment (IlkaCu) [inline src/egon/data/airflow/dags/pipeline.py:330]: Is this still needed? If not, please remove.

**Review by IlkaCu — COMMENTED** [review]
Please update the CHANGELOG.

File: src/egon/data/airflow/dags/pipeline.py (hunk 1 above)
Comment (CarlosEpia) [reply src/egon/data/airflow/dags/pipeline.py:330]: Removed.

**Comment by CarlosEpia** [comment]
The suggested changes were implemented. This branch is merged into dev.

**Pull request merged**